skl2onnx/_supported_operators.py

```diff
@@ -28,18 +28,18 @@
 from sklearn.multiclass import OneVsRestClassifier
 
 # Tree-based models
+from sklearn.ensemble import ExtraTreesClassifier
```

Comment:
import as 'alphabetic order' is recommended in PEP8?

Reply:
It's not mentioned in PEP8, but it improves readability.

Reply:
It's not mentioned in PEP8, but it improves readability.